src/class_sqlThread.py

```diff
@@ -622,3 +624,12 @@ def run(self):  # pylint: disable=too-many-locals, too-many-branches, too-many-s
 
                 helper_sql.sqlReturnQueue.put((self.cur.fetchall(), rowcount))
                 # helper_sql.sqlSubmitQueue.task_done()
+
+    def create_function(self):
+        # create_function
+        try:
+            self.conn.create_function("enaddr", 3, func=encodeAddress, deterministic=True)
+        except (TypeError, sqlite3.NotSupportedError) as err:
```

Comment:
I think this isn't correct, as xenial returns this:
`2021-03-15 11:28:56,156 - ERROR - Got error while pass deterministic in sqlite create function function takes at most 3 arguments (4 given), Passing 3 params`

Reply:
My bad. Just downgrade the log to debug severity.